sql/core/pom.xml

```diff
@@ -240,11 +241,42 @@
       <artifactId>bcpkix-jdk18on</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.github.jnr</groupId>
```

Comment:
Could we do this dependency upgrade as a separate change maybe ?

Reply:
This is needed for unix domain socket, I can make the UDS related changes to a separate PR.